### PR TITLE
[ADD] views_migration_17: module to migrate views

### PR DESCRIPTION
**Pull request by bruno-zanotti**
This is a non installable module, the aim is to use it as a server module to patch the odoo to to migrate views to Odoo version v17.

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Review by legalsylvain — COMMENTED** [review]
could you split readme.rst file into many files, as usual ?

**Comment by bruno-zanotti** [comment]
@legalsylvain done!

**Comment by bruno-zanotti** [comment]
Hey @dreispt not sure if I understand the error.. I ran the pre-commit command and I think everything is ok.. could you check?

**Comment by StefanRijnhart** [comment]
~~Wow, great work! But wouldn't this make more sense in a repo like https://github.com/OCA/odoo-module-migrator?~~
Oh never mind, that repo does not contain Odoo modules.

**Comment by StefanRijnhart** [comment]
@Bruno-Zanotti Well, pre-commit in the merge job complains about `Error: website key not found in manifest in views_migration_17.`, which is actually missing. Maybe you can make the error show up here with a rebase, but you might as well just add the `website` key to the manifest.

**Comment by bruno-zanotti** [comment]
@StefanRijnhart mm I see.. I think now should be ok!

**Comment by StefanRijnhart** [comment]
@Bruno-Zanotti  Thanks! Let's try again.

/ocabot merge nobump

**Review by zalei-odoo — COMMENTED** [review]
I liked the module. A small addition I came across when
migrate from odoo 11 to odoo 17:
` <calendar quick_add="0"> --> <calendar quick_create="0">`

File: views_migration_17/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+For eg: odoo -i upgrade_analysis -d upgrade_analysis --load=base,web,views_migration_17
+""",
+    "website": "https://github.com/OCA/server-tools",
+    "license": "AGPL-3",
+    "depends": [
+        "base",
+    ],
+    "data": [],
+    "installable": False,
```
Comment (dreispt) [inline views_migration_17/__manifest__.py:20]: This module had installable set to False, why?

Comment (bruno-zanotti) [reply views_migration_17/__manifest__.py:20]: This is a server module, the only use is to patch the odoo to migrate views to v17 (https://github.com/odoo/odoo/pull/104741). In the README it is explained how to use it.

**Comment by luisg123v** [comment]
Hi @pedrobaeza, shouldn't we add this module to the [migration wiki](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#tasks-to-do-in-the-migration)? The wiki already mentions views need to be migrated, this module could eb added as a way to automate it.

CC @desdelinux

**Comment by pedrobaeza** [comment]
Added: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0/_compare/7e3b42c665931145ccf03dfb450d312defc69b38...4c480a672acf063a72b568879c90bc04d0bf3abe

**Comment by luisg123v** [comment]
@pedrobaeza thanks a lot!

**Pull request merged**